### PR TITLE
add overlay for CDMA devices

### PR DESCRIPTION
**Pull request by nian0114**

### CODE REVIEW

**Comment by bgcngm** [comment]
Not sure if this is correct. I don't see this on stock ROM, even though it supports CDMA. I need to test this before merging it.

**Comment by yagay** [comment]
can you add fingerprint key setting please,,,,its useful to use it as home key

**Comment by bgcngm** [comment]
Merged: 0d70816fcdfd9d8d8d5e1554de99e0e71f8a9428

File: overlay/packages/services/Telephony/res/values/config.xml [hunk 1]
```diff
@@ -22,6 +22,15 @@
 <!-- Phone app resources that may need to be customized
      for different hardware or product builds. -->
 <resources>
+    <!-- Flag indicating if the phone is a world phone -->
+    <bool name="world_phone">true</bool>
```
Comment (bgcngm) [inline overlay/packages/services/Telephony/res/values/config.xml:26]: https://github.com/CyanogenMod/android_packages_services_Telephony/blob/cm-13.0/res/values/config.xml#L61

File: overlay/packages/services/Telephony/res/values/config.xml [hunk 2]
```diff
@@ -22,6 +22,15 @@
...
+
+    <!-- Flag indicating if dtmf tone type is enabled -->
+    <bool name="dtmf_type_enabled">true</bool>
```
Comment (bgcngm) [inline overlay/packages/services/Telephony/res/values/config.xml:29]: https://github.com/CyanogenMod/android_packages_services_Telephony/blob/cm-13.0/res/values/config.xml#L77

File: overlay/packages/services/Telephony/res/values/config.xml [hunk 3]
```diff
@@ -22,6 +22,15 @@
...
+
+    <!-- Show APN Settings for some CDMA carriers -->
+    <bool name="config_show_apn_setting_cdma">true</bool>
```
Comment (bgcngm) [inline overlay/packages/services/Telephony/res/values/config.xml:32]: https://github.com/CyanogenMod/android_packages_services_Telephony/blob/cm-13.0/res/values/config.xml#L190

**Pull request closed without merging**